web/src/pages/MyTransactions/TransactionDetails/PreviewCardButtons/RaiseDisputeButton.tsx

```diff
@@ -25,13 +27,24 @@ const RaiseDisputeButton: React.FC<IRaiseDisputeButton> = ({ toggleModal, button
   const { buyer, id } = useTransactionDetailsContext();
   const isBuyer = useMemo(() => address?.toLowerCase() === buyer?.toLowerCase(), [address, buyer]);
   const refetchQuery = useQueryRefetch();
+  const { data: balanceData } = useBalance({
+    address: address as `0x${string}` | undefined,
+  });
+
+  const insufficientBalance = parseFloat(arbitrationCost.toString()) > parseFloat(balanceData?.value.toString() || "0");
```

Comment:
let's use `useMemo` here
```suggestion
  const insufficientBalance = useMemo(() => {
     // assume insufficient while loading data
     if(isUndefined(arbitrationCost) || isUndefined(balanceData)) return true;
     
     return BigInt(arbitrationCost.toString()) > BigInt(balanceData.value.toString());
  },[arbitrationData, balanceData]);
```